src/app/app.cpp

```diff
@@ -64,6 +64,16 @@ int App::run(int argc, char** argv)
     qputenv("QT_STYLE_OVERRIDE", "Fusion");
     qputenv("QML_DISABLE_DISK_CACHE", "true");
 
+    //! NOTE workaround for https://musescore.org/en/node/284218
```

Comment:
Do we still need this workaround?

Reply:
I think so, otherwise we would have again the bug with emboldened fonts shown here: https://musescore.org/en/node/284218 and https://musescore.org/en/node/281601
This new workaround should address the issue with bold+underline and the issue in the Home tab preview.
Unfortunately, there is still a little problem left for large font size, see https://github.com/musescore/MuseScore/pull/17997#issuecomment-1597356396